frontend/src/app/login/MfaPhone/MfaPhone.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+
+import { Card } from "../../commonComponents/Card/Card";
+import { CardBackground } from "../../commonComponents/CardBackground/CardBackground";
+import TextInput from "../../commonComponents/TextInput";
+import Button from "../../commonComponents/Button/Button";
+
+export const MfaPhone = () => {
+  const [code, setCode] = useState("");
+  const [codeError, setCodeError] = useState("");
+
+  const validateCode = (): boolean => {
+    let error = "";
+    if (code === "") {
+      error = "Enter your security code";
+    }
+    setCodeError(error);
+    return error === "";
+  };
+
+  return (
+    <CardBackground>
+      <Card logo>
+        <h1 className="font-ui-sm margin-top-3">Enter your security code</h1>
+        <p className="usa-hint margin-bottom-0 line-height-sans-3">
+          We sent a security code to ***-***-0042. This code will expire in 10
```

Comment:
Should this be the user's phone number, instead of a default?

Reply:
Added a prop in 3729ddd